leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/servlet/ClientServlet.java

```diff
@@ -167,10 +166,6 @@ private void handleException(Exception e, HttpServletResponse resp) throws IOExc
             LOG.warn("Request rejected", e);
             resp.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
             resp.getWriter().append("Request rejected:").append(e.getMessage()).flush();
-        } else if (e instanceof RequestCanceledException) {
```

Comment:
The code is required, it's just the import that is wrong (probably due to erroneous auto completion).
What we need to import is `org.eclipse.leshan.core.request.exception.RequestCanceledException` ...